puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_hammer_plugin.yaml

```diff
@@ -0,0 +1,26 @@
+---
+- job-template:
+    name: test_hammer_plugin_{name}_{branch}
+    project-type: pipeline
+    sandbox: true
+    parameters:
+    - string:
+        name: hammer_plugin_url
+        description: 'git url of the hammer plugin repo'
+    - string:
+        name: hammer_plugin_branch
+        default: master
+        description: 'git branch of the hammer plugin to base on'
```

Comment:
These parameters are not needed.

Reply:
@ehelms, 
how can I know that the hammer git plugin will be installed?